packages/botonic-pulumi/src/index.ts

```diff
@@ -2,10 +2,27 @@ import { Config } from '@pulumi/pulumi'
 import { join } from 'path'
 import { cwd } from 'process'
 
-export const getNamePrefix = (): string => {
+export const PROJECT_NAME_SEPARATOR = '-'
+export const MAX_PROJECT_NAME_LENGTH = 30
+
+export function getNamePrefix(): string {
```

Comment:
Can we refactor this name to be more explicit too? Like `getProviderNamePrefix`

Reply:
not only used for naming the provider but for naming also the different pulumi components that are deployed. I've left it as `getProjectStackNamePrefix`